sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs

```diff
@@ -6121,6 +6188,24 @@ public partial class AutoPauseDelayTimeRange
         public Azure.ResourceManager.Sql.Models.PauseDelayTimeUnit? Unit { get { throw null; } }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct BackupStorageAccessTier : System.IEquatable<Azure.ResourceManager.Sql.Models.BackupStorageAccessTier>
```

Comment:
We should add a context as prefix of the type name of this type as well. We could simply use `Sql`

Reply:
I don't have any changes related to BackupStorage nor do I have knowledge of that, this has got autogenerated. I'll prefer to not touch that area.